lib/convertkit_v4/client.rb

```diff
@@ -0,0 +1,82 @@
+require "convertkit_v4/client/account"
+require "convertkit_v4/client/custom_fields"
+require "convertkit_v4/client/forms"
+require "convertkit_v4/client/sequences"
+require "convertkit_v4/client/subscribers"
+require "convertkit_v4/client/webhooks"
+require "convertkit_v4/client/tags"
+require "convertkit_v4/connection"
+
+module ConvertkitV4
+  class Client
+    include Account
+    include CustomFields
+    include Forms
+    include Sequences
+    include Subscribers
+    include Webhooks
+    include Tags
+
+    attr_accessor :access_token, :refresh_token, :client_id, :client_secret, :redirect_uri
+
+    AUTH_URL = "https://app.convertkit.com/oauth/authorize"
+    ACCESS_TOKEN_URL = "https://app.convertkit.com/oauth/token"
+    REVOKE_ACCESS_URL = "https://app.convertkit.com/oauth/revoke"
+
+    def initialize(access_token: nil, refresh_token: nil, client_id: nil, client_secret: nil, redirect_uri: nil)
```

Comment:
whilst we are here, can we kill two birds with one stone and support for [this](https://3.basecamp.com/5243817/buckets/25099183/card_tables/cards/6017221981)?

Basically, we should send an "integration_key" parameter for all our API requests.